plugins/view-resources/src/components/Table.svelte

```diff
@@ -230,7 +230,6 @@
     },
     {
       lookup,
-      limit: 1,
```

Comment:
Nope problem not with this. Also Total looks work fine in Table.

![image](https://github.com/hcengineering/anticrm/assets/477235/b4396a43-b789-453d-9356-06ce9064328b)

Reply:
It will request all data two times with this fix, please research why this is happening.